src/components/Inventory/withdraw/flow/SelectPurposeForm.tsx

```diff
@@ -320,6 +323,24 @@ export default function SelectPurposeForm(props: SelectPurposeFormProps): JSX.El
     setNoReadySeedlings(false);
   }, [localRecord.purpose, noReadySeedlings, snackbar, selectedNursery, OUTPLANT, batches]);
 
+  const onChangeSingleQuantity = (id: string, value: number) => {
+    setLocalRecord((previousRecord) => {
+      let modifiedBatch = {
```

Comment:
can we use some temporal values instead of this batch approach?
similar to how we do single outplant